.gitignore

```diff
@@ -13,3 +13,5 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+
+*.rac
```

Comment:
Do we really want to exclude all rac files anywhere, could we not want them as test-fixtures?